en/docs/develop/restapis/applications.yaml

```diff
@@ -0,0 +1,2962 @@
+          schema:
+            type: string
+      responses:
+        '204':
+          description: Delete Success
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '401':
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '403':
+          description: Forbidden
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '404':
+          description: Not Found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '500':
+          description: Server Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+
+
+  '/applications/{applicationId}/inbound-protocols/oidc/regenerate-secret':
+    post:
+      tags:
+        - Inbound Protocols - OAuth / OIDC
+      summary: |
+        Regenerate the OAuth2/OIDC client secret.
+      description: |
+        This API provides regenerate the OAuth2/OIDC client secret.
+      operationId: regenerateOAuthClientSecret
+      parameters:
+        - name: applicationId
+          in: path
+          description: Id of the application
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/OpenIDConnectConfiguration'
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '401':
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '403':
+          description: Forbidden
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '404':
+          description: Not Found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '500':
+          description: Server Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+
+  '/applications/{applicationId}/inbound-protocols/oidc/revoke':
+    post:
+      tags:
+        - Inbound Protocols - OAuth / OIDC
+      summary: |
+        Revokes the OAuth2/OIDC client.
+      description: |
+        This API provides revokes the OAuth2/OIDC client secret. To re-activate the client the client secret needs to be regenerated.
```

Comment:
This API **revokes** the OAuth2/OIDC...

To reactivate the client **{comma}** the client secret....